lib/ProtoPatches.rb

```diff
@@ -267,8 +267,10 @@ def add_text_style_html(text_to_insert)
       open_html_tag("a", { href: link, target: "_blank" })
     end
 
-    @active_html_node.add_child(Nokogiri::XML::Text.new(text_to_insert, @active_html_node.document))
-
+    if text_to_insert and text_to_insert.length > 0 
```

Comment:
This length check occurs prior to the deletion, but I'd like to make sure there's still text after the deletion to avoid empty tags littered about. At least in the current implementation, this function is only called with a check to make sure the text_to_insert length is greater than 0.